sympy/combinatorics/perm_groups.py

```diff
@@ -183,7 +183,7 @@ def __getitem__(self, i):
         return self._generators[i]
 
     def __contains__(self, i):
-        """Return ``True`` if `i` is contained in PermutationGroup.
+        """Return ``True`` if *i* is contained in PermutationGroup.
```

Comment:
Why does this use asterisks?

Reply:
This was a rule that we added in the new style guide based on numpydoc, that parameters are in italics (asterisks). It isn't applied in very many places yet. I think I just edited this one because I was looking at it after adding it to Sphinx.